File: desafio-03/leonelsanchesdasilva/java/desafio03.java
class Desafio03 {
  public static void main(String[] args) {
    for (int i = 1; i <= 3010; i++) {
      String numero = String.valueOf(i);
      String inverso = new StringBuilder(numero).reverse().toString();
      if (numero.equalsIgnoreCase(inverso)) {
        System.out.println(i);
      }
    }
  }
}

File: desafio-03/leonelsanchesdasilva/csharp/desafio03.cs
using System;

namespace OsProgramadores {
    internal static class StringExtensions {
        internal static string Reverse(this string s)
        {
            char[] charArray = s.ToCharArray();
            Array.Reverse(charArray);
            return new string(charArray);
        }
    }

    class Desafio03 {
        static void Main(string[] args) {
            for (int i = 1; i <= 3010; i++) {
                if (i.ToString() == i.ToString().Reverse()) {
                    Console.WriteLine(i);
                }
            }
        }
    }
}